source/04-MX/02.ptx

```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
     </statement>
 </activity>
 
-<activity estimated-time='5'>
```

Comment:
This activity computing a non-trivial kernel (which means this matrix is not invertible) is removed but the important scaffolding is implemented within the activity itself where it's needed.